README.md

```diff
@@ -6,20 +6,23 @@ A template for development with the open-autonomy framework. Find the documentat
 
 Read the requirements section [here](https://docs.autonolas.network/quick_start/).
 
-## It contains:
+## Simple ABCI example
```

Comment:
? there's none

Reply:
Please have structure here with deps, pull images, env, pull packages (even though nothing will be pulled)

Reply:
Ok, I added some more things, but I'm confused about the pull packages thing: do you mean adding some remote packages from IPFS?